.gitmodules

```diff
@@ -17,7 +17,7 @@
 [submodule "external/libtiff"]
 	path = external/libtiff
 	url = https://github.com/libsdl-org/libtiff.git
-	branch = v4.2.0-SDL
+	branch = v4.5.1-SDL
```

Comment:
I haven't ported [this patch](https://github.com/libsdl-org/libtiff/commit/2e28ca499d375be20509f82fb84dbd810ae330ba) to 4.5.1.
It should not be necessary anymore.